### ISSUE TITLE
MP trigger

### ISSUE THREAD

**Issue by zbenesch** [shdwp/dcs_liberation#32]
Hi,
Any chance you can add an "OR" between all "UNIT'S AGL ALTITUDE IS HIGHER THAN" condition?
I just tested a mission last night and the units did not trigger in MP since the "push trigger"s condition requires that ALL of the BLUE units are airborne.
I can edit the mission before the flight and remove that trigger, but if you could add the "OR" between them, it would tell dcs that any of them suffice to trigger "flag 1".
Thanks!

(Btw we are huge fans of the newest update with the routes and targets, just did yesterdays mission with one flight escort 2 flight strikers, comms on srs, we had a blast! Absolutely please do keep up the awesome job!)

**Comment by shdwp** [comment]
The point of that trigger is to have all of the units in the air. This will ensure that AI units will not trigger the actual mission start while part of the flight is still taxiing on the ground.

What is the problem with current trigger? Why do you still have units at the airport? To be completely clear, the trigger should only include aircraft that are tasked for the mission in question.

**Comment by shdwp** [comment]
Is that an issue with left over client slots, not taken by humans and, therefore, counting as "on the ground"?

**Comment by zbenesch** [comment]
> Is that an issue with left over client slots, not taken by humans and, therefore, counting as "on the ground"?

Exactly. I set out 12 hornets to be used, only 9 players showed up to fly, so 3 counted as "not being airborne".

**Comment by shdwp** [comment]
I don't see a quick fix for such issue. Will mark as "enhancement", maybe I'll add an ability to manually trigger mission start.

**Comment by zbenesch** [comment]
Thank you!
It suffices as an enhancement, seeing that single player would be most likely triggering prematurely, this would mainly benefit MP.
I can edit the triggers manually as required until I have a better idea.
Keep up the good work!

**Comment by wrycu** [comment]
I think I modified this on a fork at some point. I'll take a look at implementing it for the next release.
